### DISCUSSION TITLE
Netbeans RCP options help button

### DISCUSSION THREAD

**Discussion by highgozhangyc** [apache/netbeans#7704] [Q&A]
I developed an application using the Netbeans Rcp platform and used the shortcut page in the Netbeans options, but the page comes with a help button. How can I remove the button or add a custom event to the button?
![image](https://github.com/user-attachments/assets/87b3e09b-6a0b-46d6-8376-2185fca243c1)

**Comment by neilcsmith-net** [comment]
Removing in all places is a little challenging, but you can definitely add an override to customize what happens.  You want to look at [HelpCtx.Displayer](https://bits.netbeans.org/22/javadoc/org-openide-util-ui/org/openide/util/HelpCtx.Displayer.html).  I'm registering a simple implementation of this that opens the documentation website in one application - https://github.com/praxis-live/praxis-live/blob/master/modules/praxislive.core.ui/src/main/java/org/praxislive/ide/core/ui/OnlineHelpAction.java
